src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/Rule/RemoveAzureServiceBusRule.cs

```diff
@@ -0,0 +1,88 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Management.Automation;
+namespace Microsoft.Azure.Commands.ServiceBus.Commands.Rule
+{
+    /// <summary>
+    /// 'Remove-AzureRmServiceBusRule' Cmdlet removes the specified Rule
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, ServicebusRuleVerb, SupportsShouldProcess = true)]
```

Comment:
@v-Ajnava please make sure this cmdlet has an output type (even if it is `bool` or `void`)